package.json

```diff
@@ -28,6 +29,7 @@
     "@types/bunyan": "1.8.4",
     "body-parser": "1.18.2",
     "dotenv": "4.0.0",
+    "http-status": "1.1.0",
```

Comment:
is `http-status` required here? I know that `hadron-express` need it, but it's added there